jwst/regtest/test_nirspec_ifu.py

```diff
@@ -1,14 +1,36 @@
-"""Regression tests for NIRSpec"""
+"""Regression tests for NIRSpec IFU"""
+from pathlib import Path
 import pytest
 
+from jwst.associations import load_asn
+from jwst.lib.suffix import replace_suffix
+
 from . import regtestdata as rt
 
+# Define artifactory source and truth
+INPUT_PATH = 'nirspec/ifu'
+TRUTH_PATH = 'truth/test_nirspec_ifu'
+
 
 @pytest.fixture(scope='module')
 def run_spec2(jail, rtdata_module):
-    """Run the pipelines"""
+    """Run the Spec2Pipeline on a single exposure"""
+    rtdata = rtdata_module
+
+    # Setup the inputs
+    asn_name = 'single_nrs1_spec3_asn.json'
+    rtdata.get_data(INPUT_PATH + '/' + asn_name)
+    asn_path = rtdata.input
+    with open(asn_path, 'r') as asn_fh:
+        asn = load_asn(asn_fh)
+    member_path = Path(asn['products'][0]['members'][0]['expname'])
+    rate_path = member_path.stem
+    rate_path = replace_suffix(rate_path, 'rate')
+    rate_path = INPUT_PATH + '/' + rate_path + member_path.suffix
```

Comment:
If Artifactory expects the path separator to be '/', what is the point of using the `pathlib` module to construct part of the path for a file name on Artifactory?

Reply:
Though the current test does not have this, the test was using the local file path to get file name components. This has been removed.